src/main/java/org/folio/search/model/types/InventorySearchType.java

```diff
@@ -0,0 +1,32 @@
+package org.folio.search.model.types;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.Optional;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+
+@Getter
+@RequiredArgsConstructor
+public enum InventorySearchType {
```

Comment:
Why do you need this hardcode for TITLE? As far as I know, UI will provide the names of the fields in CQL.

Reply:
You just need to map the field name (e.g. "title"), to specific ES field(s) (e.g. "title.eng and title.original" or "title.*") having in mind what is the mapping for this field.

Reply:
it the group of fields, From my viewpoint - it is the best for now to support existing search features in inventory